2.4/Dockerfile

```diff
@@ -37,7 +34,6 @@ RUN yum install -y epel-release && \
     yum install -y --setopt=tsflags=nodocs bind-utils gettext iproute v8314 mongodb24-mongodb mongodb24 && \
     yum clean all && \
     mkdir -p /var/lib/mongodb/data && chown -R mongodb.mongodb /var/lib/mongodb/ && \
```

Comment:
chowning to 1001.1001 would defeat the purpose of saying "user 1001 isn't special"